janis_bioinformatics/tools/dawson/dawsontoolkit_0_1.py

```diff
@@ -8,8 +8,8 @@ def tool_provider(self):
 
     @staticmethod
     def container():
-        return "shollizeck/dawsontoolkit:0.1.7.1"
+        return "shollizeck/dawsontoolkit:0.1.8"
```

Comment:
Generally be careful of replacing the tool versions. It's probably better to duplicate the tool version. AFAIK , you're the only one using the tool, but it will obviously break reproducibility if you're just using `DawsonToolkit_0_1`. This won't block the release, but just something to keep in mind, especially when you change the contract of the tool.